sycl/include/CL/sycl/handler.hpp

```diff
@@ -760,9 +760,11 @@ class __SYCL_EXPORT handler {
         typename std::conditional<std::is_integral<LambdaArgType>::value &&
                                       Dims == 1,
                                   item<Dims>, LambdaArgType>::type;
+
     using NameT =
         typename detail::get_kernel_name_t<KernelName, KernelType>::name;
 
+#ifndef __SYCL_EXPLICIT_SIMD__
```

Comment:
Thanks for the quick patch. I think we all agree that this is a band-aid rather than a fix. Please add a TODO/FIXME. We also need a bug for this regression too.

Note that __SYCL_EXPLICIT_SIMD__ will soon be gone.

Reply:
> I think we all agree that this is a band-aid rather than a fix

Sure thing.

Added a FIXME.
What will be used instead of `__SYCL_EXPLICIT_SIMD__`?

Reply:
> What will be used instead of `__SYCL_EXPLICIT_SIMD__`?
We plan to unify SYCL and ESIMD modes and will get read of `-fsycl-explicit-simd` option and `__SYCL_EXPLICIT_SIMD__` macro. So, both types of kernels can coexist in the same translation unit and in the same program.